https://bugs.gentoo.org/915492
Patch from upstream branch-5-4-stable branch

commit b5b668ca442e0d7399e2b432fe5fe5fb4c48a1a2
Author: Peter Korsgaard <peter@korsgaard.com>
Date:   Sun Oct 1 10:20:31 2023 +0200

    term/post.trm: unbreak !HAVE_DEFLATE_ENCODER builds

--- a/term/post.trm
+++ b/term/post.trm
@@ -4000,17 +4000,16 @@ PS_image (unsigned int M, unsigned int N, coordval *image, gpiPoint *corner, t_i
 	cscale = 1.0;
 
 #ifdef HAVE_DEFLATE_ENCODER
-    if (ps_params->level3) {
+    if (ps_params->level3)
 	encoded_image = (void *)PS_encode_png_image(M, N, image, color_mode,
 					    bits_per_component, max_colors, cscale,
 					    &num_encoded_bytes);
-    } else {
+    else
 #endif
 	encoded_image = PS_encode_image(M, N, image, color_mode,
 					bits_per_component, max_colors, cscale,
 					(ps_params->level1 ? PS_ASCII_HEX : PS_ASCII85),
 					&num_encoded_bytes);
-    }
 
 
     fputs("%%%%BeginImage\n", gppsfile);
